index.html

```diff
@@ -353,52 +353,52 @@
-,{"system":"ClickHouse Cloud (gcp)","date":"2025-05-04","machine":"8GiB, 3 replica(s)","cluster_size":"3","comment":"","tags":["C++","column-oriented","ClickHouse derivative","managed","gcp"],"load_time":1004.566,"data_size":9948979298,"result":[[0.003,0.003,0.003],[0.194,0.134,0.023],[0.335,0.250,0.152],[0.865,0.245,0.431],[1.829,2.140,1.790],[3.610,3.269,2.792],[0.040,0.142,0.106],[0.140,0.031,0.075],[2.744,2.521,2.554],[3.108,2.991,3.500],[1.046,0.847,0.727],[1.077,0.958,0.965],[2.857,3.956,2.954],[4.366,5.303,4.584],[5.142,4.362,4.260],[2.509,2.242,2.468],[9.654,11.195,10.399],[8.426,8.433,6.442],[30.624,27.489,26.254],[0.072,0.071,0.075],[4.767,4.347,21.760],[21.331,5.052,5.117],[6.253,22.173,20.461],[19.652,63.099,20.397],[1.474,1.741,1.436],[0.971,0.990,1.024],[1.394,1.406,1.427],[4.740,4.625,4.684],[50.392,50.264,61.428],[0.156,0.158,0.174],[3.077,3.899,2.889],[4.619,8.844,3.771],[26.764,30.957,32.933],[26.534,24.458,28.214],[23.722,26.060,28.115],[1.643,1.575,1.764],[0.194,0.170,0.158],[0.494,0.057,0.056],[0.203,0.297,0.058],[0.428,0.329,0.302],[0.032,0.045,0.199],[0.159,0.020,0.020],[0.028,0.020,0.018]],"source":"clickhouse-cloud/results/gcp.3.8.json"}
+,{"system":"ClickHouse Cloud (aws)","date":"2025-05-05","machine":"12GiB, 1 replica(s)","cluster_size":"1","comment":"","tags":["C++","column-oriented","ClickHouse derivative","managed","aws"],"load_time":438.284,"data_size":9946403411,"result":[[0.002,0.002,0.002],[0.064,0.045,0.031],[0.395,0.284,0.304],[0.432,0.384,0.483],[1.864,1.706,1.781],[3.210,3.179,3.426],[0.076,0.083,0.055],[0.030,0.030,0.030],[2.489,2.395,2.359],[3.126,2.972,2.786],[0.836,0.802,0.826],[1.039,1.020,0.893],[2.575,2.527,2.296],[3.860,3.621,3.814],[3.402,3.306,3.123],[1.667,1.639,1.612],[8.739,8.920,8.495],[6.397,6.972,6.447],[23.892,23.332,23.755],[0.060,0.060,0.061],[4.863,4.907,4.722],[4.843,4.811,4.981],[5.588,5.630,5.592],[15.620,13.578,12.016],[1.075,0.946,0.959],[0.622,0.630,0.662],[0.942,0.915,1.024],[3.121,3.031,3.139],[33.148,33.494,33.080],[0.118,0.112,0.111],[1.844,1.736,1.723],[2.269,2.108,2.200],[16.095,15.594,15.364],[9.120,8.722,8.907],[8.745,8.672,8.734],[1.007,1.006,0.954],[0.089,0.093,0.092],[0.042,0.041,0.040],[0.041,0.043,0.043],[0.186,0.180,0.180],[0.025,0.023,0.021],[0.017,0.016,0.016],[0.016,0.015,0.015]],"source":"clickhouse-cloud/results/aws.1.12.json"}
```

Comment:
No need to update index.html manually, CI updates it automatically based on the results.json files.

Reply:
ah got it, pushed a commit to undo this change